staging/src/k8s.io/sample-cli-plugin/pkg/cmd/ns.go

```diff
@@ -66,7 +66,7 @@ type NamespaceOptions struct {
 // NewNamespaceOptions provides an instance of NamespaceOptions with default values
 func NewNamespaceOptions(streams genericclioptions.IOStreams) *NamespaceOptions {
 	return &NamespaceOptions{
-		configFlags: genericclioptions.NewConfigFlags(),
+		configFlags: genericclioptions.NewConfigFlags(false),
```

Comment:
Make it `true`, we want encourage people to use the new approach, and sample plugin is the place people will look.

Reply:
@soltysh Done.